tests/core/gui/test_workspaces.py

```diff
@@ -235,7 +235,7 @@ def _mounted():
         assert wk_button.button_open.isEnabled()
         assert wk_button.switch_button.isChecked()
 
-    await aqtbot.wait_until(_mounted)
+    await aqtbot.wait_until(_mounted, timeout=2000)
```

Comment:
Because it sometimes timeout, by increasing the timeout, i did not manage to reproduce the error.
https://dev.azure.com/Scille/parsec/_build/results?buildId=3611&view=logs&j=5ba45810-c0e9-5996-abe7-b2cb0c9b6baa&t=98a05ba0-83e8-5985-e9d7-1cce7a13f66e